src/components/NFTCard/NFTCard.spec.tsx

```diff
@@ -0,0 +1,225 @@
+    expect(screen.getByText('2 LAND')).toBeInTheDocument()
+  })
+})
+
+describe('when rendering a wearable', () => {
+  let wearableNFT: NFT
+
+  beforeEach(() => {
+    wearableNFT = {
+      id: 'id',
+      name: 'wearable name',
+      category: NFTCategory.WEARABLE,
+      network: Network.ETHEREUM,
+      owner: '0x1231123',
+      data: {
+        wearable: {
+          category: WearableCategory.EYES,
+          bodyShapes: [BodyShape.FEMALE],
+          isSmart: true,
+          rarity: Rarity.COMMON
+        }
+      }
+    } as NFT
+
+    screen = renderNFTCard({ nft: wearableNFT })
+  })
+
+  it('should render the wearable name', () => {
+    expect(screen.getByText(wearableNFT.name)).toBeInTheDocument()
+  })
+
+  it('should render the trimmed wearable owner', () => {
+    expect(screen.getByText('0x1231')).toBeInTheDocument()
+  })
+
+  it('should render the rarity tag', () => {
+    expect(screen.getByText(Rarity.COMMON)).toBeInTheDocument()
+  })
+
+  it('should render smart tag', () => {
+    expect(screen.getByLabelText('Smart')).toBeInTheDocument()
+  })
+})
+
+describe('when rendering an emote', () => {
+  let emoteNFT: NFT
+
+  beforeEach(() => {
+    emoteNFT = {
+      id: 'id',
+      name: 'emote name',
+      category: NFTCategory.EMOTE,
+      network: Network.ETHEREUM,
+      owner: '0x1231123',
+      data: {
+        emote: {
+          loop: true,
+          hasSound: true,
+          rarity: Rarity.COMMON,
+          category: EmoteCategory.FUN
+        }
+      }
+    } as NFT
+
+    screen = renderNFTCard({ nft: emoteNFT })
+  })
+
+  it('should render the rarity tag', () => {
+    expect(screen.getByText(Rarity.COMMON)).toBeInTheDocument()
+  })
+})
+
+describe('when a price is defined', () => {
+  let emoteNFT: NFT
+
+  beforeEach(() => {
+    emoteNFT = {
+      id: 'id',
+      name: 'emote name',
+      category: NFTCategory.EMOTE,
+      network: Network.ETHEREUM,
+      owner: '0x1231123',
+      data: {
+        emote: {
+          loop: true,
+          hasSound: true,
+          rarity: Rarity.COMMON,
+          category: EmoteCategory.FUN
+        }
+      }
+    } as NFT
+
+    screen = renderNFTCard({ nft: emoteNFT, price: '12' })
+  })
+
+  it('should render the price in MANA', () => {
+    expect(screen.getByText('12')).toBeInTheDocument()
+  })
+})
+
+describe('when a custom header is defined', () => {
+  let emoteNFT: NFT
+
+  beforeEach(() => {
+    emoteNFT = {
+      id: 'id',
+      name: 'emote name',
+      category: NFTCategory.EMOTE,
+      network: Network.ETHEREUM,
+      owner: '0x1231123',
+      data: {
+        emote: {
+          loop: true,
+          hasSound: true,
+          rarity: Rarity.COMMON,
+          category: EmoteCategory.FUN
+        }
+      }
+    } as NFT
+
+    screen = renderNFTCard({
+      nft: emoteNFT,
+      price: '12',
+      header: 'My custom header'
+    })
+  })
+
+  it('should render header text', () => {
```

Comment:
If I'm not wrong, the article is missing in these two `it`s:
```suggestion
  it('should render header text', () => {
```